### PR TITLE
Patch GDAL

### PR DESCRIPTION
**Pull request by m-chaturvedi**
Adding a few more TREs.
`nitf_spec.xml` from `2.3.2`:
https://raw.githubusercontent.com/OSGeo/gdal/v2.3.2/gdal/data/nitf_spec.xml

### CODE REVIEW

**Comment by m-chaturvedi** [comment]
+@dstoup for review, please.

File: Patches/GDAL/2.3.2/Patch.cmake [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#+
+# This file is called as CMake -P script for the patch step of
+# External_GDAL.cmake.
+# GDAL_patch and GDAL_source are defined on the command line along with the
+# call.
+#-
+
+# Fix the version number to not contain "dev" and add a SOVERSION
+# for the public C API
+message("Copying ${GDAL_patch}/frmts/mrsid/nmake.opts to ${GDAL_source}/frmts/mrsid")
```
Comment (dstoup) [inline Patches/GDAL/2.3.2/Patch.cmake]: This message seems wrong ... copy/paste from a different patch file?

Comment (dstoup) [reply Patches/GDAL/2.3.2/Patch.cmake]: Everything else looks good. I will run this through the dashboards after this fix. Assuming it passes, I will merge after.

Comment (m-chaturvedi) [reply Patches/GDAL/2.3.2/Patch.cmake]: Yes, indeed. 
Removed.

**Comment by dstoup** [comment]
Jenkins test this please

File: Patches/GDAL/2.3.2/Patch.cmake (hunk 1 above)
Comment (dstoup) [reply Patches/GDAL/2.3.2/Patch.cmake]: The comment was a good catch too, but I was speaking about the CMake message

message("Copying ${GDAL_patch}/frmts/mrsid/nmake.opts to ${GDAL_source}/frmts/mrsid")

That isn't correct either.

Comment (m-chaturvedi) [reply Patches/GDAL/2.3.2/Patch.cmake]: Sorry.

Comment (dstoup) [reply Patches/GDAL/2.3.2/Patch.cmake]: No worries, thanks.

**Pull request merged**